backend/dataall/modules/feed/services/feed_service.py

```diff
@@ -41,6 +39,15 @@ def get_feed(
         targetUri: str = None,
         targetType: str = None,
     ) -> Feed:
+        context = get_context()
+        with context.db_engine.scoped_session() as session:
+            ResourcePolicyService.check_user_resource_permission(
```

Comment:
nit: We used this logic in a couple of places already, does it worth exposing it as a decorator?

Reply:
We use it for Stacks and for Feeds no? where else do we check that is using a function in the permission_name?